charts/forseti-security/values.yaml

```diff
@@ -15,6 +15,15 @@
 # cloudsqlConnection is the connection to the cloudsql instance
 cloudsqlConnection: ""
 
+# configValidator sets whether or not to deploy config-validator
+configValidator: true
+
+# configValidatorImage is the container image used by the config-validator
+configValidatorImage: gcr.io/forseti-containers/config-validator
+
+# configValidatorImageTag is the tag for the config-validator image.
+configValidatorImageTag: latest
```

Comment:
Does this need to be a string?

Reply:
Actually, these values get interpreted as strings by default.